lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_methods.rb

```diff
@@ -171,12 +171,16 @@ def self.drb_undumped(klass)
 
     def self.ar_method
       yield
-    rescue Exception => err
+    rescue StandardError => err
```

Comment:
Keep `Exception`

Reply:
Will update, and put a rubocop disable here.